layouts/shortcodes/media-consent.html

```diff
@@ -0,0 +1,2 @@
+<h2>Media Consent</h2>
+<p>In order to take photos and videos for promotional use (mainly on our website), we are required to collect a media consent release. We therefore require everyone (including team leaders, competitors and guests) who attends the the competition to complete a media consent form. Under 18s need a parent or guardian to sign on their behalf.</p>
```

Comment:
Could we either clarify from the name of the shortcode that this is competition specific, or alternatively remove that specificity? (A couple of ways to do the latter might be to either replace "the competition" with "our events", or to pass in the name of the event as a parameter).